manual/src/html_processing/Makefile

```diff
@@ -8,8 +10,9 @@ else
     DBG=quiet
 endif
 
-WEBDIR = ../webman
-WEBDIRMAN = $(WEBDIR)/manual
+VERSION := $(OCAML_VERSION_MAJOR).$(OCAML_VERSION_MINOR)
+
+WEBDIR = ../$(VERSION)
```

Comment:
As discussed offline, I would propose to keep `WEBDIR= ../webman` and use `WEBDIRMAN =$(WEBDIR)/$(VERSION)` as the output directory. Like this we keep the structure one directory by backend while still having the manual lives under a  ($VERSION) directory.